client/components/jetpack/licensing-prompt-dialog/index.tsx

```diff
@@ -45,9 +46,13 @@ function LicensingPromptDialog( { siteId }: Props ) {
 		dispatch( recordTracksEvent( 'calypso_user_license_modal_view' ) );
 	}, [ dispatch ] );
 
-	const title = useMemo( () => {
+	let titleToRender = isEnabled( 'jetpack/pricing-page-rework-v1' )
```

Comment:
Is there a reason not to use `useMemo` anymore? Wouldn't it prevent re-calculation on every render?

Reply:
The result of that calculation is a simple string and the computation is nothing more than `if/else`. `useMemo` will be an overkill for this case.

Reply:
https://twitter.com/dan_abramov/status/1536518614110679040

<img width="598" alt="Screenshot 2022-08-22 at 5 26 28 PM" src="https://user-images.githubusercontent.com/18226415/185915760-3669d8b4-1d11-4ad6-89cb-025336c65ce7.png">

Reply:
There's also `preventWidows` but I'm not sure how heavy it is. It's not a blocker so leave it to your preference